.env.example

```diff
@@ -13,3 +13,4 @@ REGISTRY_ACCOUNT_CREATE_URL = "https://dev.bcregistry.ca/business/auth/choose-au
 KEYCLOAK_CONFIG_PATH = "keycloak-dev.json"
 AUTH_API_URL = "https://auth-api-dev.apps.silver.devops.gov.bc.ca/api/v1/"
 STATUS_API_URL = "https://status-api-dev.apps.silver.devops.gov.bc.ca/api/v1/"
+AUTH_LD_CLIENT_ID = "5db9da115f58e008123cd783"
```

Comment:
@doug-lovett This needs to be added to all environments. (It might already get imported from 1Password.)

Dev = 5db9da115f58e008123cd783
Test = 5db9d3c4076e7508aaf7db1d
Prod = 5db9d3c4076e7508aaf7db1e

Reply:
@doug-lovett ^^ This is ready for when bcregistry gets its environment variables from 1Password. But for now, I believe you will have to add this one manually to all 3 environments. Please and thanks!

Reply:
Thanks Severin, got it.